public/app/features/alerting/unified/rule-list/GrafanaGroupLoader.tsx

```diff
@@ -101,34 +96,18 @@ export function GrafanaGroupLoader({
           />
         );
       })}
-      <div className={styles.paginationWrapper}>
-        {numberOfPages > 1 && (
-          <Pagination
-            currentPage={page}
-            numberOfPages={numberOfPages}
-            onNavigate={onPageChange}
-            hideWhenSinglePage
-            className={styles.pagination}
-          />
-        )}
-      </div>
+      {hasMore && (
+        <li aria-selected="false" role="treeitem" className={styles.loadMoreWrapper}>
+          <LoadMoreButton onClick={loadMore} />
+        </li>
+      )}
     </Stack>
   );
 }
 
 const getStyles = (theme: GrafanaTheme2) => ({
-  pagination: css({
-    display: 'flex',
-    margin: 0,
+  loadMoreWrapper: css({
+    listStyle: 'none',
```

Comment:
sadly this still needs to explicitly have its list style disabled since it's not inheriting it from the parent component in the new rule group detail view.